### PR TITLE
Added Material-UI DatePicker

### LINKED ISSUE #1161: [Fix] Make the date picker accessible using keyboard keys navigation
**Describe the bug**
- The date picker does not work with the NVDA Screen Reader as it does not allow for travel to pick a date using the keyboard keys.
- The date picker should allow for keyboard keys to travel on its form fields.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'edit/create profile'
2. Click on a 'date picker field'
3. Test error

**Expected behaviour**
- The date picker should allow for the use of keyboard keys/press down to assign/pick a date to the form field. This same feature should also work in accompaniment to screen reader technologies such as NVDA.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by ianwilk20**
#### Changes introduced
- Added a Material-UI DatePicker in place of the previous Antd DatePicker 
- Added CSS and theming to the DatePicker to match the I-Talent theme
- Verified the use of tabbing (for navigation), the space bar (for making a selection), and the enter button (for confirming a selection)

#### Related issue(s)
Closes #1161

#### Screenshots (if applicable)
EducationForm
ExperienceForm
EmploymentStatusForm
SecondLanguageForm

### CODE REVIEW

File: services/frontend/src/components/formItems/DatePickerField.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { makeStyles, ThemeProvider } from "@material-ui/styles";
+import { createMuiTheme } from "@material-ui/core";
+import DayJSUtils from "@date-io/dayjs";
```
Comment (Rizvida1) [inline services/frontend/src/components/formItems/DatePickerField.jsx:9]: date-io might not have been installed properly. My compiler can't find it.

**Review by Rizvida1 — CHANGES_REQUESTED** [review]
Nice code

File: services/frontend/src/components/formItems/DatePickerField.jsx (hunk 1 above)
Comment (ianwilk20) [reply services/frontend/src/components/formItems/DatePickerField.jsx:9]: Not sure what caused this... 

I restarted my local server and it got rid of the compiler error that said: "Module not found". 

I removed the date-io package and re-added it with its dependencies. Please let me know if this did anything

**Review by Rizvida1 — APPROVED** [review]
Slightly hard to use but works so great job!

**Pull request merged**